src/Mentat.UI/Views/Card/AddEditCard.cshtml

```diff
@@ -34,7 +36,8 @@
             </div>
             <div class="form-group">
                 <label asp-for="DifficultyLevel" class="control-label"></label>
-                <input asp-for="DifficultyLevel" class="form-control" />
+                @Html.DropDownListFor(m => m.DifficultyLevel, ((DifficultyLevel[]) Enum.GetValues(typeof(DifficultyLevel))).ToList()
+                    .Select(d => new SelectListItem { Text = d.ToString(), Value = d.ToString() } ), null, new { style = "width: 250px;" })
```

Comment:
If difficulty level were a string[], this could be simplified to be 

```csharp
Html.DropDownListFor(m => m.DifficultyLevel, string[] FlashCardDifficultyLevels.Select(difficulty => new SelectListItem(text: difficulty, value: difficulty) ...
```

It frees up a lot of the conversion involved when turning an Enum to a string and having to iterate over the enum values using reflection. That said, not a required change. 

At a minimum, you can remove the ToList()

Reply:
Since the professor is good with me leaving it as an Enum, resolving this comment.